internal/monitors/zcodegen/codegen.go

```diff
@@ -0,0 +1,6 @@
+// Package codegen exists purely to manipulate the go:generate build order so
+// that collectd templates are generated first before the monitor metadata
+// module is generated.
+package codegen
+
+//go:generate bash -c "go build -o ./monitorcodegen ../../../cmd/monitorcodegen && ./monitorcodegen"
```

Comment:
This will require having bash installed on Windows right?

Reply:
Yeah I missed switching the windows script to use go generate.  Will fix.

Reply:
Ok, fixed it to work with windows and to make windows CI actually use go generate.